app/uk/gov/hmrc/personaldetailsvalidationfrontend/personaldetails/PersonalDetailsCollectionController.scala

```diff
@@ -14,23 +14,24 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.personaldetailsvalidationfrontend.helloworld
+package uk.gov.hmrc.personaldetailsvalidationfrontend.personaldetails
 
 import javax.inject.Inject
 
 import play.api.i18n.{I18nSupport, MessagesApi}
-import play.api.mvc._
+import play.api.mvc.{Action, AnyContent}
 import uk.gov.hmrc.personaldetailsvalidationfrontend.config.ViewConfig
+import uk.gov.hmrc.personaldetailsvalidationfrontend.model.JourneyId
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 
 import scala.concurrent.Future
 
-class HelloWorldEndpoint @Inject()(page: HelloWorldPage)
-                                  (implicit val messagesApi: MessagesApi, viewConfig: ViewConfig)
+class PersonalDetailsCollectionController @Inject()(page: PersonalDetailsPage)
```

Comment:
MessagesApi, ViewConfig, I18nSupport can be mixed in/ injected into the PersonalDetailsPage. 

Might make controller lighter.